password.php

```diff
@@ -29,6 +29,7 @@
 require_once(dirname(__FILE__).'/locallib.php');
 require_once($CFG->libdir.'/tcpdf/tcpdf_barcodes_2d.php'); // Used for generating qrcode.
 
+
```

Comment:
hmm - I would have expected this change would have triggered a travis failure - we're not supposed to have 2 empty lines in the code - when you get a chance can you please remove this extra line - thanks!